### PR TITLE
Add Ethan as a collaborator

### PR DESCRIPTION
**Pull request by rachelmangan**
Second test pull request

### CODE REVIEW

File: collaborators.md [hunk 1]
```diff
@@ -1,3 +1,4 @@
 * Rachel Moore
 * Pawan Poudel
-* Yuri Shevtsov
\ No newline at end of file
+* Yuri Shevtsov
```
Comment (pawanpoudel) [inline collaborators.md:3]: Oops... looks like we forgot to enable the "add new line on save" feature on VS Code.

Comment (rachelmangan) [reply collaborators.md:3]: Good catch, I have enabled the save for future commits to avoid this going forward.

**Comment by pawanpoudel** [comment]
jfksdfjkdsjfkdsfds

File: collaborators.md [hunk 2]
```diff
@@ -1,4 +1,5 @@
 * Rachel Moore
```
Comment (pawanpoudel) [inline collaborators.md:1]: Some insightful comment about the overall design of this file.

**Comment by pawanpoudel** [comment]
Everything looks good! 👍 Please go ahead and merge it into master. :shipit:

**Pull request merged**